bitnami/elasticsearch/values.yaml

```diff
@@ -1729,35 +2044,96 @@ metrics:
   ## Prometheus Operator ServiceMonitor configuration
   ##
   serviceMonitor:
-    ## @param metrics.serviceMonitor.enabled if `true`, creates a Prometheus Operator ServiceMonitor (also requires `metrics.enabled` to be `true`)
+    ## @param metrics.serviceMonitor.enabled Create ServiceMonitor Resource for scraping metrics using PrometheusOperator
     ##
     enabled: false
-    ## @param metrics.serviceMonitor.namespace Namespace in which Prometheus is running
+    ## @param metrics.serviceMonitor.namespace Namespace which Prometheus is running in
+    ## e.g:
+    ## namespace: monitoring
     ##
     namespace: ""
-    ## @param metrics.serviceMonitor.labels Extra labels for the ServiceMonitor
+    ## @param metrics.serviceMonitor.jobLabel The name of the label on the target service to use as the job name in prometheus.
     ##
```

Comment:
These variables are declared in the template yaml but not present here: 

```
Key coordinating.extraEnvVarsConfigMap not found
Key coordinating.podManagementPolicy not found
Key coordinating.service.port not found
Key coordinating.terminationGracePeriodSeconds not found
Key curator.dryrun not found
Key curator.env not found
Key curator.envFromSecrets not found
Key curator.jobAnnotations not found
Key curator.jobLabels not found
Key curator.securityContext not found
Key data.extraEnvVarsConfigMap not found
Key extraEnvVarsConfigMap not found
Key ingest.extraEnvVarsConfigMap not found
Key ingest.ingress.extrarules not found
Key ingest.ingress.ingressClassName not found
Key ingress.extrarules not found
Key ingress.ingressClassName not found
Key master.extraEnvVarsConfigMap not found
Key metrics.args not found
Key metrics.command not found
Key metrics.containerSecurityContext not found
Key metrics.containerSecurityContext.enabled not found
Key metrics.customLivenessProbe not found
Key metrics.customReadinessProbe not found
Key metrics.extraEnvVars not found
Key metrics.extraEnvVarsCM not found
Key metrics.extraEnvVarsSecret not found
Key metrics.extraVolumeMounts not found
Key metrics.extraVolumes not found
Key metrics.fullnameOverride not found
Key metrics.initContainers not found
Key metrics.podSecurityContext not found
Key metrics.podSecurityContext.enabled not found
Key metrics.priorityClassName not found
Key metrics.service.port not found
Key metrics.sidecars not found
Key security.fipsMode not found
Key security.tls.elasticPassword not found
Key security.tls.passwordsSecret not found
```

Some of them may be false positives